test/libsolidity/SemanticTest.cpp

```diff
@@ -46,77 +46,102 @@ SemanticTest::SemanticTest(string const& _filename, string const& _ipcPath, lang
 	m_source = parseSourceAndSettings(file);
 	if (m_settings.count("compileViaYul"))
 	{
-		m_validatedSettings["compileViaYul"] = m_settings["compileViaYul"];
-		m_compileViaYul = true;
-		m_settings.erase("compileViaYul");
+		if (m_settings["compileViaYul"] == "also")
+		{
+			m_validatedSettings["compileViaYul"] = m_settings["compileViaYul"];
+			m_runWithYul = true;
+			m_runWithoutYul = true;
+		}
+		else
+		{
```

Comment:
Could assert that `m_settings["compileViaYul"] == "only"`

Reply:
Then we'd need to update all of them now, though :-)... my plan was to postpone that until we're sure we stay with this syntax...